requirements/edx/base.txt

```diff
@@ -633,6 +633,10 @@ lazy==1.4
     #   acid-xblock
     #   lti-consumer-xblock
     #   ora2
+learner-pathway-progress==1.2
```

Comment:
how the plugin is installed? i couldnt find name of plugin in `base.in` file.

Reply:
Goot catch.